assets/javascripts/discourse/widgets/discourse-reactions-actions.js

```diff
@@ -168,6 +168,8 @@ export default createWidget("discourse-reactions-actions", {
   },
 
   touchStart() {
+    this._validTouch = true;
```

Comment:
Does this definitely work as expected? IIRC there's no guarantee that `this` will refer to the same Widget instance for each event (touch start/move/end).